cypress/integration/rectangle.spec.js

```diff
@@ -158,5 +158,36 @@ describe('Draw Rectangle', () => {
       const coords = geojson.geometry.coordinates;
       expect(coords.length).to.equal(1);
     })
-  })
+  });
+
+  it('disable popup on layer while drawing', ()=>{
+    let rect = null;
+    cy.window().then(({ map, L }) => {
+      map.on("pm:create",(e)=>{
+        e.layer.bindPopup('Popup test');
+        if(e.layer instanceof L.Rectangle){
+          rect = e.layer;
+        }
+      });
+    });
+
+    cy.toolbarButton('rectangle').click();
+    cy.get(mapSelector)
+      .click(100,50)
+      .click(700,400);
+
```

Comment:
is this test really testing if the popup is disabled during through? I would expect a click on the layer here and then check that the popup is NOT there

Reply:
When the existing rectangle has a popup and we want to place a marker on it, it opens the popup instead of adding the marker. So it tests if the marker is placed and existing